Filtered_kernel/include/CGAL/internal/Static_filters/Collinear_3.h

```diff
@@ -76,33 +76,14 @@ class Collinear_3
       else if (max1 > upper_bound_1)
         upper_bound_1 = max1;
 
-      int int_tmp_result;
-      if (lower_bound_1 < 5.00368081960964635413e-147)
-        return Base::operator()(p, q, r);
-      else
-      {
-        if (upper_bound_1 > 1.67597599124282407923e+153)
-          return Base::operator()(p, q, r);
-
+      if ((lower_bound_1 >= 5.00368081960964635413e-147) && (upper_bound_1 <= 1.67597599124282407923e+153)) {
         double eps = (8.88720573725927976811e-16 * (max1 * max2));
-        if (double_tmp_result > eps)
-          int_tmp_result = 1;
-        else
-        {
-          if (double_tmp_result < -eps)
-            int_tmp_result = -1;
-          else
-            return Base::operator()(p, q, r);
-        }
+        if ((double_tmp_result > eps) || (double_tmp_result < -eps))
```

Comment:
you changed the next one to use abs()>eps but not this one? (not CGAL::abs so it would use sse2fabs on windows?)
we could move the computation of double_tmp_result just before this line, though it probably doesn't change anything, so not necessary.